src/java.desktop/unix/classes/sun/print/CUPSPrinter.java

```diff
@@ -92,6 +94,13 @@ public Void run() {
         libFound = initIDs();
         if (libFound) {
            cupsServer = getCupsServer();
```

Comment:
I think we should wrap all the new lines in isMac()
Also can you explain the reasons for the logic that implies we may have a server starting with "/"
in which case your always change the cupServer to localhost even if it is not sandboxed ?

I ask because it seems to contradict what you pasted
"by the cupsServer() function and not changing the interface string to "localhost""